.circleci/config.yml

```diff
@@ -1,17 +1,64 @@
-version: 2
+version: 2.1
 jobs:
-  "test":
+  "build-test":
     docker:
-      - image: docker.io/golang:1.17.0
+      - image: alpine:3.16
     steps:
       - checkout
+      - run:
+          name: Install deps
+          command: apk add gcc musl-dev make bash git go
+      - run:
+          name: Build
+          command: make build
       - run:
           name: Test
           command: make test
+      - run:
+          name: Prepare Artifacts
+          command: |
+            mkdir -p /tmp/build-amd64
+            cp _output/bin/driverkit /tmp/build-amd64/
+      - persist_to_workspace:
+          root: /tmp
+          paths:
+            - build-amd64/
+  "build-test-arm64":
+    machine:
+      enabled: true
+      image: ubuntu-2004:2022.04.1
+    resource_class: arm.medium
+    steps:
+      - checkout:
+          path: /tmp/source
+      - run:
+          name: Prepare project
+          command: |
+            docker run --rm -it -v /tmp/source:/source -w /source --name alpine_sh -d alpine:3.16 sh
+            docker exec alpine_sh apk add gcc musl-dev make bash git go
+            docker exec alpine_sh git config --global --add safe.directory /source
```

Comment:
This is needed because workspace cloned `/tmp/source` and docker `/source` are owned by different users.